packages/airnode-protocol/contracts/rrp/requesters/RrpBeaconServer.sol

```diff
@@ -297,7 +297,7 @@ contract RrpBeaconServer is
         onlyIfTemplateExists(templateId)
         returns (bool)
     {
-        return userIsWhitelisted(templateId, reader);
+        return userIsWhitelisted(templateId, reader) || reader == address(0);
```

Comment:
I considered doing this in `Whitelist.sol` inside `userIsWhitelisted()` but that feels like a bad idea

Reply:
I thought about doing the same but allowing `address(0)` to read the beacon seems like an edge case of RrpBeaconServer.sol and Whitelist.sol should take a more generic approach IMO.